lepton/package.go

```diff
@@ -49,25 +49,123 @@ func DownloadPackage(name string) (string, error) {
 
 	archivename := name + ".tar.gz"
 	packagepath := path.Join(PackagesCache, archivename)
-	if _, err := os.Stat(packagepath); os.IsNotExist(err) {
-		if err = DownloadFileWithProgress(packagepath,
-			fmt.Sprintf(PackageBaseURL, archivename), 600); err != nil {
+	_, err = os.Stat(packagepath)
+	if err != nil && !os.IsNotExist(err) {
+		return "", err
+	}
+
+	if err == nil {
+		return packagepath, nil
+	}
+
+	pkgBaseURL := PackageBaseURL
+
+	// Check config override
+	if config != nil {
+		cPkgBaseURL := strings.Trim(config.PackageBaseURL, " ")
+		if len(cPkgBaseURL) > 0 {
+			pkgBaseURL = cPkgBaseURL
+		}
+	}
+
+	// Check environment variable override
+	ePkgBaseURL := os.Getenv("OPS_PACKAGE_BASE_URL")
+	if len(ePkgBaseURL) > 0 {
```

Comment:
not a big deal, but usually the idiom ePkgBaseURL != "" is typically preferred

Reply:
ok, fixed by https://github.com/nanovms/ops/pull/1068/commits/df339a12c9f4a0dd3f31453d1ad63413883ebf6d